templates/web/base/report/new/duplicate_suggestions.html

```diff
@@ -6,9 +6,33 @@
     <button class="duplicate-report-suggestions__close js-hide-duplicate-suggestions">[% loc('Close') %]</button>
     <h2 class="form-section-heading">[% loc('Already been reported?') %]</h2>
     <div class="form-section-description">
-        <p>[% loc('We’ve already sent these similar nearby reports to the council. You can track them to receive updates.') %]</p>
+        <p>[% loc('We’ve already reported these similar problems nearby reports to the council. Is one of them yours?') %]</p>
     </div>
     <ul class="item-list"></ul>
-    <button class="btn btn--block js-hide-duplicate-suggestions">[% loc('No, continue with my new report') %]</button>
+    <button class="btn btn--block js-hide-duplicate-suggestions">[% loc('Continue – report a new problem') %]</button>
+</div>
+<div class="js-template-get-updates hidden">
+    <div id="alerts" class="get-updates">
+        <p id="rznvy_hint">
+            <strong>[% loc('Great!') %]</strong>
+          [% IF c.user_exists %]
+            [% loc('Would you like us to notify you when this problem is updated or fixed?') %]
+          [% ELSE %]
+            [% loc('If you let us know your email address, we’ll notify you when this problem is updated or fixed.') %]
+          [% END %]
```

Comment:
This is fine, but I think it could say the logged in text either way and it'd be clear enough?

Reply:
Do you think it’ll be obvious enough, for a first-time user, without an account, that when they see `Would you like us to notify you when this problem is updated or fixed?` and a `Your email` input box below it, that they should enter their email address in the box to get updates?

I mean, when I explain it like that, of course it feels obvious enough. But I’m a bit worried that the answer to "Would you like us to notify/email you?" is a yes/no, which is fine when it’s followed by just a button, but a bit weird when it’s followed by a text input. What do you think?

Reply:
I dunno. I'm happy either way I think.